backend/endpoints/volunteer.py

```diff
@@ -0,0 +1,54 @@
+from flask.views import MethodView
+from flask import jsonify, request
+from models import Volunteer
+from config import PassHash, MIN_PASSWORD_LEN
+
+
+class VolunteerAPI(MethodView):
+
+    def get(self, volunteer_id:str):
+        try:
+            if volunteer_id:
+                volunteer = Volunteer.objects(id=volunteer_id).get().clean_data()
+                return jsonify(volunteer)
+            else:
+                volunteers = [v.clean_data() for v in Volunteer.objects(is_active=True).all()]
+                return jsonify({"list": volunteers})
+        except Exception as error:
```

Comment:
Do you know which specific exceptions might occur here? If so, it would be better to catch them specifically, instead of having a broad exception handler.